tests/beaker_tests/cisco_overlay_global/test_overlay_global.rb

```diff
@@ -103,8 +102,12 @@
 
 tests['preclean'] = {
   manifest_props: '',
-  code:           [0, 2],
-  resource_props: {},
+  code:           [0],
+  resource_props: {
+    # These properties always exist
+    'dup_host_mac_detection_host_moves' => '5',
+    'dup_host_mac_detection_timeout'    => '180',
+  },
 }
 
 tests['default_properties'] = {
```

Comment:
Added. Also wrapped the command_config calls in preclean with this check as I suspect they will probably fail on other platforms.